src/components/ClipboardAwarePasteButton.tsx

```diff
@@ -17,7 +17,9 @@ export default function ClipboardAwarePasteButton({
   const { t } = useTranslation()
 
   useLayoutEffect(() => {
-    LayoutAnimation.easeInEaseOut()
+    if (Platform.OS !== 'android') {
```

Comment:
i think this is our only option...reading the [LayoutAnimation docs](https://reactnative.dev/docs/0.66/layoutanimation) the first paragraph already calls out that `Note that in order to get this to work on Android you need to set the following flags via UIManager` but we've explicitly disabled this in the past https://github.com/valora-inc/wallet/blob/83630ce2878cde47431d966380e0c3c3f179a40a/src/app/App.tsx#L62-L67

Reply:
Yes we RN's builtin LayoutAnimation is quite buggy on Android and I've been burned a couple of times trying to re-enable it.